main.js

```diff
@@ -1,20 +1,46 @@
 const coffeeShop = {
   beans: 40,
 
+  beanPrice: 0.2,
+  
+  money: 5000,
+
   drinkRequirements: {
-    latte: 10,
-    americano: 5,
-    doubleShot: 15,
-    frenchPress: 12
+    latte: { beanRequirements: 10, price: 5 },
+    americano: { beanRequirements: 5, price: 3 },
+    doubleShot: { beanRequirements: 15, price: 8 },
+    frenchPress: { beanRequirements: 12, price: 7 }
   },
 
   makeDrink: function (drinkType) {
-    // TODO: Finish this method
+    const toMake = this.drinkRequirements[drinkType]
```

Comment:
variable names should be indicative of the value they hold.
`toMake` doesn't say much
`drinkToMake` would be better
just `drink` is enough